lib/mage/Readme.md

```diff
@@ -76,7 +76,11 @@ only argument.
 
 ### mage.quit(number exitCode)
 
-Shuts down MAGE and exits the process with the given exit code, or 0 if none has been passed.
+Shuts down MAGE and all its processes.
```

Comment:
Might be nice to mention that the master process will exit with the given exit code.

Reply:
The readme's should go at some point... so TBH I don't really want to spend too much time on them.